From: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Subject: mm: drop inappropriate includes
Date: Tue, 23 Jul 2024 18:11:48 +0100

As pointed out by SeongJae Park, these includes are not always available to
all architectures and in actual fact are not required to be included here.

Link: https://lkml.kernel.org/r/c1c009dd-1ee3-4615-a8d1-b1971f22a814@lucifer.local
Signed-off-by: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Reported-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vma_internal.h |    2 --
 1 file changed, 2 deletions(-)

--- a/mm/vma_internal.h~mm-move-internal-core-vma-manipulation-functions-to-own-file-fix
+++ a/mm/vma_internal.h
@@ -43,8 +43,6 @@
 #include <linux/userfaultfd_k.h>
 
 #include <asm/current.h>
-#include <asm/page_types.h>
-#include <asm/pgtable_types.h>
 #include <asm/tlb.h>
 
 #include "internal.h"
_
